CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,6 @@
+- Complete the Provisioning API with CRUD Operations.
```

Comment:
Before merging this PR I'd reserach a bit how changelog and version managing works in NPM. We should find the equivalent to the CHANGES_NEXT_RELESE "flush" process into .spec changelog in the NPM world.